src/RedisEnterpriseCache/examples/Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink.md

```diff
@@ -0,0 +1,6 @@
+### Example 1: Unlink database
+```powershell
+Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink -ResourceGroupName "MyGroup" -ClusterName "MyCache3" -Id @("<database-id>")
```

Comment:
please remove /< and > from example . They are special keywords in powershell example.
```suggestion
Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink -ResourceGroupName "MyGroup" -ClusterName "MyCache3" -Id @("database-id")
```